language/fr/pages_common.php

```diff
@@ -39,5 +39,5 @@
 //
 
 $lang = array_merge($lang, array(
-	'PAGES_VIEWONLINE'		=> 'Consulte %s',
+	'PAGES_VIEWONLINE'		=> 'Consulte la page “%s”',
```

Comment:
I have added "la page" because in the Official FRENCH translation of phpBB it is also use those terms to specify "what we are talking ?", for example : 
```
	'READING_FORUM'				=> 'Consulte les sujets dans %s',
	'READING_GLOBAL_ANNOUNCE'	=> 'Consulte une annonce globale',
	'READING_LINK'				=> 'Consulte le lien du forum %s',
	'READING_TOPIC'				=> 'Consulte un sujet dans %s',
```
I'm not a robot, i want to make some sense. The english language is not the same as french.